README.tmpl.md

```diff
@@ -110,6 +110,12 @@ go install -ldflags "-X 'main.version=$(git describe --tags)' -X 'main.commit=$(
 
 On Windows, you can run the above commands with Git Bash, which comes with [Git for Windows](https://git-scm.com/download/win).
 
+### GitHub Actions installation
+
+To use GolangCI Lint as GitHub action in your repo please visit [GolangCI Lint Action](https://github.com/matoous/golangci-lint-action).
+and click on `View on Marketplace`. You can also take a look how to add badge from this action into the README.md in the
```

Comment:
Where `View in Marketplace` supposed to be? I can't find this any button/link ☹️

Reply:
![102227F2-65B8-4D3A-8890-6787951AA03E](https://user-images.githubusercontent.com/15747583/65555241-53438600-df2c-11e9-9b4d-db6de607b180.jpeg)

Reply:
Changed the README to link directly to the marketplace.

Reply:
After f2f conversation we realized, that not all GitHub users can see the button on the page with the repo.